app/services/archivesspace.service.js

```diff
@@ -96,7 +57,7 @@ factory('ArchivesSpace', ['Restangular', function(Restangular) {
       },
       list_digital_object_component_contents: function(id, component_id) {
         var id_fragment = id_to_urlsafe(id);
-        return ArchivesSpace.one(id_fragment).one('digital_object_components').one(String(component_id)).one('files').get().then(decode_entry_response).then(format_results);
+        return ArchivesSpace.one(id_fragment).one('digital_object_components').one(String(component_id)).one('files').get().then(decode_browse_response).then(format_entries);
```

Comment:
Are parent_path and parent getting passed to format_entries, or are they supposed to be undefined/null?